### PR TITLE
Align Integrations Documentation Structure

### PR DESCRIPTION
**Pull request by snaselj**
# Closes: NaN

## What's Changed

- Aligned integrations documentation structure with SSoT.

### CODE REVIEW

**Comment by snaselj** [comment]
@cmsirbu Could you pls. check this?

File: docs/admin/integrations/aristacv.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ This guide will walk you through steps to set up Arista CloudVision integration
 
 Before configuring the integration, please ensure the following:
 
-- `nautobot-chatops` App was [installed with integration extra dependencies](./index.md#installation-guide).
+- `nautobot-chatops` App was [installed with integration extra dependencies](./../install.md#installation-guide).
```
Comment (cmsirbu) [inline docs/admin/integrations/aristacv.md:9]:
```suggestion
- `nautobot-chatops` App was [installed with integration extra dependencies](../install.md#installation-guide).
```

File: docs/admin/integrations/aristacv.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ This guide will walk you through steps to set up Arista CloudVision integration
...
     ```shell
     pip install nautobot-chatops[aristacv]
     ```
-- `nautobot-chatops` App is set up with at least one [enabled chat platform](./index.md#chat-platforms-configuration) and [tested](./index.md#test-your-chatbot).
+- `nautobot-chatops` App is set up with at least one [enabled chat platform](./../install.md#chat-platforms-configuration) and [tested](./../install.md#test-your-chatbot).
```
Comment (cmsirbu) [inline docs/admin/integrations/aristacv.md:13]:
```suggestion
- `nautobot-chatops` App is set up with at least one [enabled chat platform](../install.md#chat-platforms-configuration) and [tested](./../install.md#test-your-chatbot).
```

File: docs/admin/integrations/aristacv.md [hunk 3]
```diff
@@ -6,15 +6,15 @@ This guide will walk you through steps to set up Arista CloudVision integration
...
 
 ## Command Setup
 
-Create a top-level command named `cloudvision` in your enabled chat platform. For detailed instructions related to your specific chat platform, refer to the [platform specific set up](./index.md#chat-platforms-configuration).
+Create a top-level command named `aristacv` in your enabled chat platform. For detailed instructions related to your specific chat platform, refer to the [platform specific set up](./../install.md#chat-platforms-configuration).
```
Comment (cmsirbu) [inline docs/admin/integrations/aristacv.md:17]:
```suggestion
Create a top-level command named `aristacv` in your enabled chat platform. For detailed instructions related to your specific chat platform, refer to the [platform specific set up](../install.md#chat-platforms-configuration).
```

File: docs/admin/integrations/grafana.md [hunk 4]
```diff
@@ -6,18 +6,18 @@ This guide will walk you through steps to set up Grafana integration with the `n
 
 Before configuring the integration, please ensure the following:
 
-- `nautobot-chatops` App was [installed with integration extra dependencies](./index.md#installation-guide).
+- `nautobot-chatops` App was [installed with integration extra dependencies](./../install.md#installation-guide).
```
Comment (cmsirbu) [inline docs/admin/integrations/grafana.md:9]:
```suggestion
- `nautobot-chatops` App was [installed with integration extra dependencies](../install.md#installation-guide).
```

File: docs/admin/integrations/grafana.md [hunk 5]
```diff
@@ -6,18 +6,18 @@ This guide will walk you through steps to set up Grafana integration with the `n
...
     ```shell
     pip install nautobot-chatops[grafana]
     ```
-- `nautobot-chatops` App is set up with at least one [enabled chat platform](./index.md#chat-platforms-configuration) and [tested](./index.md#test-your-chatbot).
+- `nautobot-chatops` App is set up with at least one [enabled chat platform](./../install.md#chat-platforms-configuration) and [tested](./../install.md#test-your-chatbot).
```
Comment (cmsirbu) [inline docs/admin/integrations/grafana.md:13]:
```suggestion
- `nautobot-chatops` App is set up with at least one [enabled chat platform](../install.md#chat-platforms-configuration) and [tested](./../install.md#test-your-chatbot).
```

File: docs/admin/integrations/grafana.md [hunk 6]
```diff
@@ -6,18 +6,18 @@ This guide will walk you through steps to set up Grafana integration with the `n
...
 - [Grafana](https://grafana.com/docs/grafana/latest/installation/) application installed and configured with dashboards and panels.
 - [Grafana Image Rendering Service](https://grafana.com/docs/grafana/latest/administration/image_rendering/) installed.
 - [Grafana Image Rending App for Grafana](https://grafana.com/grafana/plugins/grafana-image-renderer/) installed in your Grafana application.
 
 ## Command Setup
 
-Create a top-level command named `grafana` in your enabled chat platform. For detailed instructions related to your specific chat platform, refer to the [platform specific set up](./index.md#chat-platforms-configuration).
+Create a top-level command named `grafana` in your enabled chat platform. For detailed instructions related to your specific chat platform, refer to the [platform specific set up](./../install.md#chat-platforms-configuration).
```
Comment (cmsirbu) [inline docs/admin/integrations/grafana.md:20]:
```suggestion
Create a top-level command named `grafana` in your enabled chat platform. For detailed instructions related to your specific chat platform, refer to the [platform specific set up](../install.md#chat-platforms-configuration).
```

File: docs/admin/integrations/ipfabric.md [hunk 7]
```diff
@@ -6,11 +6,11 @@ This guide will walk you through steps to set up IPFabric integration with the `
 
 Before configuring the integration, please ensure the following:
 
-- `nautobot-chatops` App was [installed with integration extra dependencies](./index.md#installation-guide).
+- `nautobot-chatops` App was [installed with integration extra dependencies](./../install.md#installation-guide).
```
Comment (cmsirbu) [inline docs/admin/integrations/ipfabric.md:9]:
```suggestion
- `nautobot-chatops` App was [installed with integration extra dependencies](../install.md#installation-guide).
```

File: docs/admin/integrations/ipfabric.md [hunk 8]
```diff
@@ -6,11 +6,11 @@ This guide will walk you through steps to set up IPFabric integration with the `
...
     ```shell
     pip install nautobot-chatops[ipfabric]
     ```
-- `nautobot-chatops` App is set up with at least one [enabled chat platform](./index.md#chat-platforms-configuration) and [tested](./index.md#test-your-chatbot).
+- `nautobot-chatops` App is set up with at least one [enabled chat platform](./../install.md#chat-platforms-configuration) and [tested](./../install.md#test-your-chatbot).
```
Comment (cmsirbu) [inline docs/admin/integrations/ipfabric.md:13]:
```suggestion
- `nautobot-chatops` App is set up with at least one [enabled chat platform](../install.md#chat-platforms-configuration) and [tested](./../install.md#test-your-chatbot).
```

File: docs/admin/integrations/ipfabric.md [hunk 9]
```diff
@@ -25,7 +25,7 @@ Here is a compatibility matrix and the minimum versions required to run this app
 
 ## Command Setup
 
-Create a top-level command named `ipfabric` in your enabled chat platform. For detailed instructions related to your specific chat platform, refer to the [platform specific set up](./index.md#chat-platforms-configuration).
+Create a top-level command named `ipfabric` in your enabled chat platform. For detailed instructions related to your specific chat platform, refer to the [platform specific set up](./../install.md#chat-platforms-configuration).
```
Comment (cmsirbu) [inline docs/admin/integrations/ipfabric.md:28]:
```suggestion
Create a top-level command named `ipfabric` in your enabled chat platform. For detailed instructions related to your specific chat platform, refer to the [platform specific set up](../install.md#chat-platforms-configuration).
```

File: docs/admin/integrations/meraki.md [hunk 10]
```diff
@@ -6,15 +6,15 @@ This guide will walk you through steps to set up Cisco Meraki integration with t
 
 Before configuring the integration, please ensure the following:
 
-- `nautobot-chatops` App was [installed with integration extra dependencies](./index.md#installation-guide).
+- `nautobot-chatops` App was [installed with integration extra dependencies](./../install.md#installation-guide).
```
Comment (cmsirbu) [inline docs/admin/integrations/meraki.md:9]:
```suggestion
- `nautobot-chatops` App was [installed with integration extra dependencies](../install.md#installation-guide).
```

File: docs/admin/integrations/meraki.md [hunk 11]
```diff
@@ -6,15 +6,15 @@ This guide will walk you through steps to set up Cisco Meraki integration with t
...
     ```shell
     pip install nautobot-chatops[meraki]
     ```
-- `nautobot-chatops` App is set up with at least one [enabled chat platform](./index.md#chat-platforms-configuration) and [tested](./index.md#test-your-chatbot).
+- `nautobot-chatops` App is set up with at least one [enabled chat platform](./../install.md#chat-platforms-configuration) and [tested](./../install.md#test-your-chatbot).
```
Comment (cmsirbu) [inline docs/admin/integrations/meraki.md:13]:
```suggestion
- `nautobot-chatops` App is set up with at least one [enabled chat platform](../install.md#chat-platforms-configuration) and [tested](./../install.md#test-your-chatbot).
```

**Review by cmsirbu — CHANGES_REQUESTED** [review]
Left a bunch of minor fixes, great work otherwise! Once the build errors are fixed, we should be good to go.

Path has an extra `..` which causes build errors in all 3 includes in this file:

```
ERROR   -  Error reading page 'admin/install.md':
ERROR   -  No files found including '../../glossary.md' at admin/install.md:16
```

**Comment by cmsirbu** [comment]
```
--- a/docs/admin/install.md
+++ b/docs/admin/install.md
@@ -13,7 +13,7 @@ This guide outlines the process of enabling Nautobot ChatOps, which includes:
   - [Test Your Chatbot](#test-your-chatbot)
   - [Integrations Configuration](#integrations-configuration)

-{% include-markdown '../../glossary.md' heading-offset=1 %}
+{% include-markdown '../glossary.md' heading-offset=1 %}

 ## Prerequisites

@@ -126,7 +126,7 @@ Adjust the App's behavior with the following settings:
 ## Granting Access to the Chat Platform

 {%
-    include-markdown '../../models/accessgrant.md'
+    include-markdown '../models/accessgrant.md'
     start='<!--access-grant-->'
     heading-offset=1
 %}
@@ -134,7 +134,7 @@ Adjust the App's behavior with the following settings:
 ## Link Nautobot Account

 {%
-    include-markdown '../../models/chatopsaccountlink.md'
+    include-markdown '../models/chatopsaccountlink.md'
     start='<!--account-link-->'
     heading-offset=1
 %}
```

**Comment by snaselj** [comment]
> ```
> --- a/docs/admin/install.md
> +++ b/docs/admin/install.md
> @@ -13,7 +13,7 @@ This guide outlines the process of enabling Nautobot ChatOps, which includes:
>    - [Test Your Chatbot](#test-your-chatbot)
>    - [Integrations Configuration](#integrations-configuration)
> 
> -{% include-markdown '../../glossary.md' heading-offset=1 %}
> +{% include-markdown '../glossary.md' heading-offset=1 %}
> 
>  ## Prerequisites
> 
> @@ -126,7 +126,7 @@ Adjust the App's behavior with the following settings:
>  ## Granting Access to the Chat Platform
> 
>  {%
> -    include-markdown '../../models/accessgrant.md'
> +    include-markdown '../models/accessgrant.md'
>      start=''
>      heading-offset=1
>  %}
> @@ -134,7 +134,7 @@ Adjust the App's behavior with the following settings:
>  ## Link Nautobot Account
> 
>  {%
> -    include-markdown '../../models/chatopsaccountlink.md'
> +    include-markdown '../models/chatopsaccountlink.md'
>      start=''
>      heading-offset=1
>  %}
> ```

Fixed

**Comment by snaselj** [comment]
This pull request has been refreshed and is ready for review.

**Review by cmsirbu — APPROVED** [review]

**Comment by smk4664** [comment]
I hope you don't mind, I fixed another mkdocs issue, unrelated to your pull request.

**Review by smk4664 — APPROVED** [review]

**Pull request merged**